used for applying patches when building bundled mdb library; we use
databases/lmdb from ports instead

Index: pyproject.toml
--- pyproject.toml.orig
+++ pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel", "cffi>=0.8", "patch-ng"]
+requires = ["setuptools", "wheel", "cffi>=0.8"]
 build-backend = "setuptools.build_meta"
 
 [tool.cibuildwheel]
